nxdrive/gui/application.py

```diff
@@ -313,6 +324,7 @@ def action_progressing(self, action: Action) -> None:
             log.warning(f"An action is needed, got {action!r}")
             return
         self.transfer_model.set_progress(action.export())
+        self.direct_transfer_model.set_progress(action.export())
```

Comment:
We should have a way to make the difference. `action.export()` should be stored into a variable and determine if this is a DT or not to not call `.set_progress` on all models.